docs/components/clerk-provider.mdx

```diff
@@ -81,80 +85,17 @@ ReactDOM.render(
+| `navigate?`               | `(to: string) => Promise<unknown> \| void` | A function which takes the destination path as an argument and performs a "push" navigation.                                                                                                                               |
+| `clerkJSUrl?`             | `string`                                   | Define the URL that `@clerk/clerk-react` should hot-load `@clerk/clerk-js` from.                                                                                                                                           |
+| `clerkJSVariant?`         | `string`                                   | If your web application only uses Control components you can set this value to `headless` and load a minimal ClerkJS bundle for optimal page performance.                                                                  |
+| `clerkJSVersion?`         | `string`                                   | Define the npm version for `@clerk/clerk-js`.                                                                                                                                                                              |
+| `supportEmail?`           | `string`                                   | Optional support email for display in authentication screens. Will only affect Clerk Components and not Account Portal pages.                                                                                              |
+| `localization?`           | `object`                                   | Optional object to localize your Components. Will only affect Clerk Components and not Account Portal pages.                                                                                                               |
+| `appearance?`             | `object`                                   | Optional object to customize your Components. Will only affect Clerk Components and not Account Portal pages.                                                                                                              |
+| `allowedRedirectOrigins?` | `Array<string \| RegExp>`                  | Optional array of domains used to validate against the query param of an auth redirect. If no match is made, the redirect is considered unsafe and the default redirect will be used with a warning passed to the console. |
+| `afterSignInUrl`          | `string`                                   | Full URL or path to navigate after successful sign in.                                                                                                                                                                     |
+| `afterSignUpUrl`          | `string`                                   | Full URL or path to navigate after successful sign up.                                                                                                                                                                     |

```

Comment:
@LekoArts Should we add `isSatellite`, `proxyUrl`, and `domain` ?

Reply:
Yes, we should. But our internal docs are sparse on this and call it experimental so I left it out. But feel free to put up a PR or nudge the person who would know more to also add it in another PR :)

Reply:
We reference `isSatellite` and `domain` on https://clerk.com/docs/advanced-usage/satellite-domains so we can grab the info from there and add it here for consistency if nothing else.

Reply:
@LekoArts I've updated the codebase and removed the experimental tags.